### PR TITLE
🌏 convert cram to gvcf to public app

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

This PR creates a public app for the cram to gvcf workflow. It also updates the readme to include the link to the public app and pulls in the pipeline images directly from github so they populate the public app.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Pushed to apps-publisher and it pulls the files appropriately: https://cavatica.sbgenomics.com/u/cavatica/apps-publisher/apps/#cavatica/apps-publisher/kfdrc-gatk-haplotypecaller-workflow

**Test Configuration**:
* Environment: Cavatica
* Test files: None

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: workflows/kfdrc-gatk-haplotypecaller-wf.cwl [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+id: kfdrc-gatk-haplotypecaller-workflow
+label: Kids First DRC GATK HaplotypeCaller CRAM to gVCF Workflow
+doc: |
+  ## KFDRC GATK HaplotypeCaller CRAM to gVCF Workflow
+
+  This workflow taks a CRAM file, converts it to a BAM, determines a contamination value, then runs
+  GATK HaplotypeCaller to generate a gVCF, gVCF calling metrics, and, if no contamination value is provided,
+  the VerifyBAMID output.
+
+  This workflow is the current production workflow, equivalent to this [Cavatica public app](https://cavatica.sbgenomics.com/public/apps#cavatica/apps-publisher/kfdrc-gatk-haplotypecaller-workflow)
```
Comment (migbro) [inline workflows/kfdrc-gatk-haplotypecaller-wf.cwl:12]: Huh? There is one already? or is it the chicken and egg issue or readme vs doc?

File: workflows/kfdrc-gatk-haplotypecaller-wf.cwl [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+class: Workflow
+id: kfdrc-gatk-haplotypecaller-workflow
+label: Kids First DRC GATK HaplotypeCaller CRAM to gVCF Workflow
+doc: |
+  ## KFDRC GATK HaplotypeCaller CRAM to gVCF Workflow
+
+  This workflow taks a CRAM file, converts it to a BAM, determines a contamination value, then runs
+  GATK HaplotypeCaller to generate a gVCF, gVCF calling metrics, and, if no contamination value is provided,
+  the VerifyBAMID output.
+
```
Comment (migbro) [inline workflows/kfdrc-gatk-haplotypecaller-wf.cwl:11]: Please add in our fancy pants logo

File: workflows/kfdrc-gatk-haplotypecaller-wf.cwl [hunk 3]
```diff
@@ -0,0 +1,215 @@
+cwlVersion: v1.0
+class: Workflow
+id: kfdrc-gatk-haplotypecaller-workflow
+label: Kids First DRC GATK HaplotypeCaller CRAM to gVCF Workflow
+doc: |
+  ## KFDRC GATK HaplotypeCaller CRAM to gVCF Workflow
```
Comment (migbro) [inline workflows/kfdrc-gatk-haplotypecaller-wf.cwl:6]: Can drop this header as the `label` gets displayed anyway

**Review by migbro — CHANGES_REQUESTED** [review]
Some minor change requests

File: workflows/kfdrc-gatk-haplotypecaller-wf.cwl (hunk 1 above)
Comment (dmiller15) [reply workflows/kfdrc-gatk-haplotypecaller-wf.cwl:12]: Chicken/egg shenanigans.

File: workflows/kfdrc-gatk-haplotypecaller-wf.cwl (hunk 2 above)
Comment (dmiller15) [reply workflows/kfdrc-gatk-haplotypecaller-wf.cwl:9]: Added

File: workflows/kfdrc-gatk-haplotypecaller-wf.cwl (hunk 3 above)
Comment (dmiller15) [reply workflows/kfdrc-gatk-haplotypecaller-wf.cwl]: Done

File: workflows/kfdrc-gatk-haplotypecaller-wf.cwl [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+id: kfdrc-gatk-haplotypecaller-workflow
+label: Kids First DRC GATK HaplotypeCaller CRAM to gVCF Workflow
+doc: |
+  This workflow taks a CRAM file, converts it to a BAM, determines a contamination value, then runs
+  GATK HaplotypeCaller to generate a gVCF, gVCF calling metrics, and, if no contamination value is provided,
+  the VerifyBAMID output.
+
+  ![data service logo](https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png)
+
+  This workflow is the current production workflow, equivalent to this [Cavatica public app](https://cavatica.sbgenomics.com/public/apps#cavatica/apps-publisher/kfdrc-gatk-haplotypecaller-workflow)
```
Comment (migbro) [inline workflows/kfdrc-gatk-haplotypecaller-wf.cwl:12]: redundant, points back to app in this context

**Review by migbro — CHANGES_REQUESTED** [review]

**Review by migbro — APPROVED** [review]

**Review by sickler-alex — APPROVED** [review]

**Pull request merged**